chef/cookbooks/cpe_utils/libraries/node_functions.rb

```diff
@@ -356,5 +368,39 @@ def installed?(bundle_identifier)
       paths = app_paths(bundle_identifier)
       !paths.empty?
     end
+
+    def min_package_installed?(pkg_identifier, min_pkg_version)
+      unless macos?
+        Chef::Log.warn('node.min_package_installed? called on non-OS X!')
+        return false
+      end
+      installed_pkg_version = shell_out(
+        "/usr/sbin/pkgutil --pkg-info \"#{pkg_identifier}\"",
+      ).run_command.stdout.to_s[/version: (.*)/, 1]
+      # Compare the installed version to the minimum version
+      if Gem::Version.new(installed_pkg_version) >= Gem::Version.new(
```

Comment:
you dont need that block. remove the if and return true/false. 

`Gem::Version.new(installed_pkg_version) >= Gem::Version.new(min_pkg_version)`

Reply:
also you can same some cols with shortening the vars
`Gem::Version.new(installed_pkg) >= Gem::Version.new(min_pkg)`

Reply:
explain your first comment. Shortening the vars just seems like OCD on your part, but I will change them.